README.md

```diff
@@ -320,6 +324,46 @@ In all other cases there will be an error status:
 }
 ```
 
+#### Configure wait-online reboot timeout
```

Comment:
do we want to document, that we test for `--any` per default and `OMNECT_WAIT_ONLINE_INTERFACES` can alter this configuration? resp. provide this as optional parameter in this method?

Reply:
> do we want to document, that we test for `--any` per default and OMNECT_WAIT_ONLINE_INTERFACES can alter this configuration?

yes

> resp. provide this as optional parameter in this method?

would this really be a use case? I propose to create a "to be discussed task"